workflow-runtime/src/commonTest/kotlin/com/squareup/workflow1/internal/SubtreeManagerTest.kt

```diff
@@ -172,10 +174,34 @@ internal class SubtreeManagerTest {
     assertFalse(tickOutput.isCompleted)
 
     eventHandler("event!")
-    val update = tickOutput.await().value!!
+    val update = tickOutput.await().output!!.value!!
 
-    val (_, output) = update.applyTo("props", "state")
-    assertEquals("case output:workflow output:event!", output?.value)
+    val (_, result) = update.applyTo("props", "state")
+    assertEquals("case output:workflow output:event!", result.output!!.value)
+    assertFalse(result.stateChanged)
+  }
+
+  @Test fun tick_children_handles_no_child_output() = runTest {
```

Comment:
nit / out of scope: we don't have a `tick()` method any more, should we update test and var names here?

Reply:
Good point. #1012